use_cases/eluc/prescriptors/esp/create_seeds.py

```diff
@@ -0,0 +1,138 @@
+    """
+
+    no_change_preds = df[constants.RECO_COLS].copy()
+    y_train = no_change_preds.to_numpy()
+    X_train = [encoded_df[col].values for col in constants.CAO_MAPPING["context"]]
+
+    no_change_model = create_template_model()
+    opt = tf.keras.optimizers.legacy.Adam(learning_rate=0.001)
+    no_change_model.compile(optimizer=opt, loss='mean_absolute_error', metrics=['mae'])
+    no_change_model.fit(X_train, y_train, epochs=n_epochs, batch_size=4096, verbose=1)
+
+    seed_dir.mkdir(parents=True, exist_ok=True)
+    no_change_model.save(seed_dir / "1_1.h5")
+
+def seed_max_change(seed_dir: Path, df: pd.DataFrame, encoded_df: pd.DataFrame, n_epochs=300, best_col="secdf"):
+    """
+    Creates seed model that attempts to prescribe maximum change.
+    Moves all possible land use to best_col which is secdf by default.
+    """
+    # Move all the land use to secdf
+    land_use = df[constants.RECO_COLS].sum(axis=1)
+    max_change_preds = df[constants.RECO_COLS].copy()
+    max_change_preds[constants.RECO_COLS] = 0
+    max_change_preds[best_col] = land_use
+
+    y_train = max_change_preds.to_numpy()
+    X_train = [encoded_df[col].values for col in constants.CAO_MAPPING["context"]]
+    
+    max_change_model = create_template_model()
+    opt = tf.keras.optimizers.legacy.Adam(learning_rate=0.001)
+    max_change_model.compile(optimizer=opt, loss='mean_absolute_error', metrics=['mae'])
+    max_change_model.fit(X_train, y_train, epochs=n_epochs, batch_size=4096, verbose=1)
+
+    seed_dir.mkdir(parents=True, exist_ok=True)
+    max_change_model.save(seed_dir / "1_2.h5")
+
+def validate_seeds(seed_dir: Path, nn_path: Path, presc_cfg_path:Path, dataset: ELUCData):
+    """
+    TODO: This is pretty yucky right now and exposes some internals in the dummy prescriptor,
+    will have to play around with the SWE side of the prescriptors to make this work better.
+    Validates that the seeds' performances match the intended behavior.
+    Creates a dummy prescriptor and evaluates the seeds, then prints the results.
+    """
+    nnp = NeuralNetPredictor()
+    nnp.load(nn_path)
+    with open(presc_cfg_path) as f:
+        presc_config = json.load(f)
+    dummy_prescriptor = UnileafPrescriptor(presc_config,
+                                    dataset.train_df.iloc[:1],
+                                    dataset.encoder,
+                                    [nnp])
+    
+    test_df = dataset.test_df.sample(frac=0.01, random_state=100)
+    context_df = test_df[constants.CAO_MAPPING["context"]]
+
+    for seed_path in seed_dir.iterdir():
+        candidate = load_model(seed_path)
+        encoded_context_df = dataset.encoder.encode_as_df(context_df)
+        reco_land_use = dummy_prescriptor.prescribe(candidate, encoded_context_df)
+        reco_df = pd.DataFrame(reco_land_use["reco_land_use"].tolist(), columns=constants.RECO_COLS)
+        context_actions_df = dummy_prescriptor._reco_to_context_actions(reco_df, encoded_context_df)
```

Comment:
We call _reco_to_context_actions here because there is currently no way to call prescribe_and_predict without the underlying assumption that the file structure will be the same as how the results are saved during training (as opposed to seed training)